Classes/Player/Tracks/PKCaptionsAssetResourceLoaderDelegate.swift

```diff
@@ -16,7 +16,6 @@ class PKCaptionsAssetResourceLoaderDelegate: NSObject, AVAssetResourceLoaderDele
     
     private let extM3UPrefix = "#EXTM3U"
```

Comment:
I don't think these constants should be instance fields. Either static fields or in the file itself, as `fileprivate` (preferred).

Reply:
Created a struct with all the ones we are using.